2021-10-05 22:12:59,046 INFO    MainThread:3242 [wandb_setup.py:_flush():69] setting env: {}
2021-10-05 22:12:59,046 INFO    MainThread:3242 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '8bbaf831241b8525faf8ba0890cc99d295161186'}
2021-10-05 22:12:59,046 INFO    MainThread:3242 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '8bbaf831241b8525faf8ba0890cc99d295161186'}
2021-10-05 22:12:59,046 INFO    MainThread:3242 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-05 22:12:59,046 INFO    MainThread:3242 [wandb_init.py:_log_setup():342] Logging user logs to /root/konst/ml-utils/training/ner_employment/wandb/run-20211005_221259-3nki8dc8/logs/debug.log
2021-10-05 22:12:59,046 INFO    MainThread:3242 [wandb_init.py:_log_setup():343] Logging internal logs to /root/konst/ml-utils/training/ner_employment/wandb/run-20211005_221259-3nki8dc8/logs/debug-internal.log
2021-10-05 22:12:59,047 INFO    MainThread:3242 [wandb_init.py:init():375] calling init triggers
2021-10-05 22:12:59,047 INFO    MainThread:3242 [wandb_init.py:init():382] wandb.init called with sweep_config: {}
config: {'data_path': '/root/konst/data/bio_dataset/bio_ner_dataset.csv', 'checkpoint_path': '/root/konst/model_checkpoints/ner_employment', 'model_path': '', 'batch_size': 128, 'epochs': 40, 'start_epoch': 0, 'test_size': 0.05, 'lam': 0, 'patience': 5, 'factor': 0.5, 'use_wandb': True, 'lr': 0.00015, 'save_at': 1, 'adjust_lr_every': 1, 'num_workers': 0}
2021-10-05 22:12:59,047 INFO    MainThread:3242 [wandb_init.py:init():424] starting backend
2021-10-05 22:12:59,047 INFO    MainThread:3242 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-05 22:12:59,061 INFO    MainThread:3242 [backend.py:ensure_launched():135] starting backend process...
2021-10-05 22:12:59,066 INFO    MainThread:3242 [backend.py:ensure_launched():140] started backend process with pid: 3257
2021-10-05 22:12:59,067 INFO    MainThread:3242 [wandb_init.py:init():429] backend started and connected
2021-10-05 22:12:59,069 INFO    MainThread:3242 [wandb_init.py:init():477] updated telemetry
2021-10-05 22:12:59,070 INFO    MainThread:3242 [wandb_init.py:init():500] communicating current version
2021-10-05 22:12:59,546 INFO    MainThread:3242 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-10-05 22:12:59,546 INFO    MainThread:3242 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-10-05 22:12:59,757 INFO    MainThread:3242 [wandb_init.py:init():540] starting run threads in backend
2021-10-05 22:13:00,522 INFO    MainThread:3242 [wandb_run.py:_console_start():1584] atexit reg
2021-10-05 22:13:00,522 INFO    MainThread:3242 [wandb_run.py:_redirect():1458] redirect: SettingsConsole.REDIRECT
2021-10-05 22:13:00,523 INFO    MainThread:3242 [wandb_run.py:_redirect():1463] Redirecting console.
2021-10-05 22:13:00,524 INFO    MainThread:3242 [wandb_run.py:_redirect():1519] Redirects installed.
2021-10-05 22:13:00,524 INFO    MainThread:3242 [wandb_init.py:init():565] run started, returning control to user process
2021-10-05 23:30:46,696 INFO    MainThread:3242 [wandb_run.py:_atexit_cleanup():1554] got exitcode: 1
2021-10-05 23:30:46,697 INFO    MainThread:3242 [wandb_run.py:_restore():1526] restore
2021-10-05 23:30:49,009 INFO    MainThread:3242 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1139
  total_bytes: 1139
}

2021-10-05 23:30:49,221 INFO    MainThread:3242 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1139
  total_bytes: 1139
}

2021-10-05 23:30:50,073 INFO    MainThread:3242 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  uploaded_bytes: 1139
  total_bytes: 13997
}

2021-10-05 23:30:50,176 INFO    MainThread:3242 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1139
  total_bytes: 15525
}

2021-10-05 23:30:50,278 INFO    MainThread:3242 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9875
  total_bytes: 15525
}

2021-10-05 23:30:50,380 INFO    MainThread:3242 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 15525
  total_bytes: 15525
}

2021-10-05 23:30:50,483 INFO    MainThread:3242 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 15525
  total_bytes: 15525
}

2021-10-05 23:30:50,586 INFO    MainThread:3242 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 15525
  total_bytes: 15525
}

2021-10-05 23:30:50,690 INFO    MainThread:3242 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 15525
  total_bytes: 15525
}

2021-10-05 23:30:51,068 INFO    MainThread:3242 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 15525
  total_bytes: 15525
}

2021-10-05 23:30:51,171 INFO    MainThread:3242 [wandb_run.py:_wait_for_finish():1676] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 15525
  total_bytes: 15525
}

2021-10-05 23:30:52,569 INFO    MainThread:3242 [wandb_run.py:_show_summary():1831] rendering summary
2021-10-05 23:30:52,570 INFO    MainThread:3242 [wandb_run.py:_show_history():1869] rendering history
2021-10-05 23:30:52,571 INFO    MainThread:3242 [wandb_run.py:_show_files():1898] logging synced files
